Source/Meadow.Core.Samples/Network/Cell_Basics/Cell_Basics.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Meadow.Sdk/1.1.0">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <OutputType>Library</OutputType>
+    <AssemblyName>App</AssemblyName>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Meadow.F7" Version="0.*" />
```

Comment:
Change version to `Version="0.*"`, otherwise, it'll pick up the last version before we went 1.0